src/themes/themes.less

```diff
@@ -0,0 +1,26 @@
+//This is an auto-generated themes definition file. Do not edit manually!
+//To make changes during development, run `npm run start` or
+//`npm run storybook`
```

Comment:
I'm not a fan of having auto-generated files be part of git. Not saying it should be removed, but wanted your opinion on this.

Reply:
I've seen this argued both ways, with some folks even committing node_modules to the repo. In this context though, I can remove it from the repo pretty easily. Will do!